locales/zh_CN/LC_MESSAGES/gui.po

```diff
@@ -94,176 +96,184 @@ msgid ""
-msgstr ""
+msgstr "（可选，RGB十六进制）颜色填充圆圈"
 
 #: ../../gui.rst:93 ../../gui.rst:103
 msgid "(optional, scalar) the radius of circle"
-msgstr ""
+msgstr "（可选，标量）圆的半径"
 
 #: ../../gui.rst:95
 msgid "Draw a solid circle."
-msgstr ""
+msgstr "画一个实心圆。"
 
 #: ../../gui.rst:101
 msgid "(np.array) the position of circles"
-msgstr ""
+msgstr "(np.array) 圆的位置"
 
 #: ../../gui.rst:102
 msgid "(optional, RGB hex or np.array of uint32) color(s) to fill circles"
-msgstr ""
+msgstr "（可选，uint32的RGB十六进制或np.array）颜色以填充圆圈"
 
 #: ../../gui.rst:105
 msgid "Draw solid circles."
-msgstr ""
+msgstr "画多个实心圆。"
 
 #: ../../gui.rst:109
 msgid ""
 "If ``color`` is a numpy array, circle at ``pos[i]`` will be colored with "
 "``color[i]``, therefore it must have the same size with ``pos``."
 msgstr ""
+"如果 ``color`` 是一个numpy数组，则 ``pos[i]`` 的圆圈将使用 ``color[i]`` 进行着"
+"色，因此它的大小必须与 ``pos`` 相同。"
 
 #: ../../gui.rst:115
 msgid "(tuple of 2) the first end point position of line"
-msgstr ""
+msgstr "（2元组）直线的第一个端点位置"
 
 #: ../../gui.rst:116
 msgid "(tuple of 2) the second end point position of line"
-msgstr ""
+msgstr "（2元组）直线的第二个端点位置"
 
 #: ../../gui.rst:117
 msgid "(optional, RGB hex) the color of line"
-msgstr ""
+msgstr "（可选，RGB十六进制）线条颜色"
 
 #: ../../gui.rst:118
 msgid "(optional, scalar) the width of line"
-msgstr ""
+msgstr "（可选，标量）线的宽度"
 
 #: ../../gui.rst:120
 msgid "Draw a line."
-msgstr ""
+msgstr "画一条线。"
 
 #: ../../gui.rst:126
 msgid "(tuple of 2) the first end point position of triangle"
-msgstr ""
+msgstr "画一条线。"
 
 #: ../../gui.rst:127
 msgid "(tuple of 2) the second end point position of triangle"
-msgstr ""
+msgstr "（2元组）三角形的第二个端点位置"
 
 #: ../../gui.rst:128
 msgid "(tuple of 2) the third end point position of triangle"
-msgstr ""
+msgstr "（2个元组）三角形的第三个端点位置"
 
 #: ../../gui.rst:129
 msgid "(optional, RGB hex) the color to fill the triangle"
-msgstr ""
+msgstr "（可选，RGB十六进制）填充三角形的颜色"
 
 #: ../../gui.rst:131
 msgid "Draw a solid triangle."
-msgstr ""
+msgstr "画一个实心三角形。"
 
 #: ../../gui.rst:137
 msgid "(tuple of 2) the top-left point position of rectangle"
-msgstr ""
+msgstr "（2元组）矩形的左上角位置"
 
 #: ../../gui.rst:138
 msgid "(tuple of 2) the bottom-right point position of rectangle"
-msgstr ""
+msgstr "（2元组）矩形的右下角位置"
 
 #: ../../gui.rst:139
 msgid "(optional, RGB hex) the color of stroke line"
-msgstr ""
+msgstr "（可选，RGB十六进制）描边线的颜色"
 
 #: ../../gui.rst:140
 msgid "(optional, scalar) the width of stroke line"
-msgstr ""
+msgstr "（可选，标量）描边线的宽度"
 
 #: ../../gui.rst:142
 msgid "Draw a hollow rectangle."
-msgstr ""
+msgstr "画一个空心的矩形。"
 
 #: ../../gui.rst:146
 msgid "Event processing"
-msgstr ""
+msgstr "事件处理"
 
 #: ../../gui.rst:148
 msgid ""
-"Every event have a key and type. *Event key* is the key that you pressed "
-"on keyboard or mouse, can be one of:"
+"Every event have a key and type. *Event key* is the key that you pressed on "
+"keyboard or mouse, can be one of:"
 msgstr ""
+"每个事件都有一个键和类型。 *Event key* 是您在键盘或鼠标上按下的键，可以是以下之"
+"一："
```

Comment:
```suggestion
"每个事件都有一个 ``key`` 和 ``type``。 ``key`` 是您在键盘或鼠标上按下的键，可以是以下之"
"一："
```
Let's keep `key` and `type` untranslated, since they are the attribute of class `Event`.